pkg/operator/k8sutil/k8sutil.go

```diff
@@ -75,9 +76,10 @@ func GetK8SVersion(clientset kubernetes.Interface) (*version.Version, error) {
 	return version.MustParseSemantic(serverVersion.GitVersion), nil
 }
 
-// Hash MD5 hash a given string
+// StableName computes a stable pseudorandom string suitable for inclusion in a Kubernetes object name from the given seed string.
```

Comment:
nit: The first word in the comment should still be `Hash` since that's the name of the method

Reply:
I was suggesting to change the name of the method as well.